docs/src/examples/dual_quaternions.md

```diff
@@ -88,8 +88,18 @@ function transform(d::DualQuaternion, p::AbstractVector)
     return pnew
 end
 
+function rotmatrix_from_quat(q::Quaternion)
```

Comment:
Minor point, but I prefer we avoid `_from_quat` unless necessary, since in Julia, `_from_XXX` is handled by the type constraint in the method overload, so it's redundant.

Reply:
I agree with that as the common manner in the Julia language. However this is just documentation and readers cannot check the methods of the function by running `methods(XXX)`, and I thought explicit `XXX_from_YYY` is a little better.